src/css-module-dts-loader/loader.ts

```diff
@@ -43,11 +43,18 @@ function generateDTSFile(filePath: string, sourceFilesRegex: RegExp): Promise<vo
 			const css = cssMap.get(filePath) || '';
 			mTimeMap.set(filePath, mtime);
 
-			if (newCss !== css) {
+			if (newCss !== css || !definition) {
```

Comment:
Tries to create a definition if there isn't one already, so that we can create the empty export to support empty css files.